lib/licensee/project.rb

```diff
@@ -45,6 +34,19 @@ def license
       @license ||= license_file.match if license_file
     end
 
+    # Scores a given file as a potential license
+    #
+    # filename - (string) the name of the file to score
+    #
+    # Returns 1 if the file is definately a license file
+    # Return 0.5 if the file is likely a license file
+    # Returns 0 if the file is definately not a license file
+    def self.match_license_file(filename)
+      return 1   if self.license_file?(filename)
```

Comment:
Should the match score be higher if the filename matches a file closer to the beginning of LICENSE_FILENAMES array? It seems that the names in that array are sorted by preference.